oadr/oadr/manager/VENManager.cpp

```diff
@@ -200,6 +207,9 @@ void VENManager::poll()
 		// processMessage(response->oadrSignedObject().oadrCancelPartyRegistration().get());
 		// TODO: what's the proper response when a cancel registration message is received?
 
+		// oadrCanceledPartyRegistration()?
```

Comment:
Accidentally, I found an answer to the question in the above TODO. I haven't fixed that yet to reduce the scope of this PR.